tasks/verify-enterprise-contract/0.1/verify-enterprise-contract.yaml

```diff
@@ -95,9 +95,12 @@ spec:
       default: "now"
 
   results:
-    - name: HACBS_TEST_OUTPUT
+    - name: TEST_OUTPUT
       description: Short summary of the policy evaluation for each image
 
+    - name: HACBS_TEST_OUTPUT
+      description: Short summary of the policy evaluation for each image (Deprecated)
+
```

Comment:
Perhaps we don't reorder in case someone consumes this as `results[0]` and then expects the `name` to be `HACBS_TEST_OUTPUT`

Reply:
Good idea. Done.